sql/moz-fx-data-shared-prod/mozilla_vpn_derived/users_v1/query.sql

```diff
@@ -1,6 +1,133 @@
+WITH raw AS (
+  SELECT
+    fxa_uid,
+    created_at,
+    JSON_EXTRACT(attribution, '$') AS attribution,
```

Comment:
pending #1585 this can be decoded via:
```suggestion
    mozfun.json.extract_string_map(attribution) AS attribution,
```

Reply:
committed this, to see if CI passes

Reply:
Ah, I've got to get format-sql working first...